packages/bento-design-system/src/SliderField/createSliderField.tsx

```diff
@@ -42,11 +48,26 @@ export function createSliderField({
             onChange: (values: number[]) => props.onChange(values[0]),
           };
 
-    const state = useSliderState({
+    const _state = useSliderState({
```

Comment:
super minor, ma possiamo usare un qualsiasi altro nome che non inizi con `_`? Usando `_` TS ignora i warning di unused, quindi se per caso diventa unused in un refactor non ce ne accorgiamo